sdk/turing/router/config/experiment_config.py

```diff
@@ -29,9 +29,9 @@ def config(self) -> Dict:
 
     @config.setter
     def config(self, config: Dict):
-        if config is not None and 'project_id' in config:
-            config['project_id'] = int(config['project_id'])
         self._config = config
+        if self._config is not None and 'project_id' in self._config:
+            self.config['project_id'] = int(self._config['project_id'])
```

Comment:
Same issue as the one I just commented on, though the fix here was to simply reorder the statements.